modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -45,11 +45,12 @@ To decouple the minimum Mule version when running tests, add the `-Dmunit.jvm` p
 Before upgrading and testing your custom connector for Java 17, you must be familiar with the following considerations:
 
 * Target compilation level
-From Mule 4.9.0 and onwards, connectors can be compiled with any Java versions providing it can generate bytecode the proper version.
 +
-For this, the version of `mule-modules-parent` must be upgraded to 1.9.0 of higher, and the `java.release.version` set to the minimal Java version that the connector will require. (See xref:java-version-support.adoc#bytecode_level[Bytecode Level Configuration]).
+In Mule 4.9.0 and later, you can compile any Mule project for Java 17 or lower. You must also compile any additional dependencies for Java 17 or lower. 
```

Comment:
`You must also compile ` -> `You may compile `

Reply:
actually, it should be `Additional dependencies must be compiled for Java 17 or lower` since 3pp dependencies are already pre-compiled